pkg/model/storage.go

```diff
@@ -32,6 +35,8 @@ const (
 	S3
 )
 
+var validS3LogLevels = []string{"OFF", "FATAL", "ERROR", "WARN", "INFO", "DEBUG", "TRACE"}
```

Comment:
We need to tell endusers what are the valid levels. Maybe add them to error message and/or swagger doc?